### PR TITLE
Add a ChangeDBCluster script

### PR DESCRIPTION
**Pull request by paladox**
Allows you to update multiple wikis and set their db cluster to a different value.

### CODE REVIEW

File: maintenance/changeDBCluster.php [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+		parent::__construct();
+		$this->addOption( 'file', 'Path to file where the wikinames are store. Must be one wikidb name per line. (Optional, fallsback to current dbname)', false, true );
+		$this->addOption( 'db-cluster', 'Sets the wikis requested to a different db cluster.', true, true );
+	}
+
+	public function execute() {
+		global $wgCreateWikiDatabase, $wgCreateWikiDatabaseClusters, $wgDBname;
+		
+		if ( !$wgCreateWikiDatabaseClusters ) {
+			$this->fatalError( "You have not set \$wgCreateWikiDatabaseClusters." );
```
Comment (JohnFLewis) [inline maintenance/changeDBCluster.php:22]: Is this strictly a requirement? I don't see it being used elsewhere in the script.

Comment (JohnFLewis) [reply maintenance/changeDBCluster.php:22]: Also if you used '' instead of "", you'd not need to escape

Comment (paladox) [reply maintenance/changeDBCluster.php:22]: Well, no not strictly required i guess.

I'll remove, thanks!

File: maintenance/changeDBCluster.php [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+			],
+			__METHOD__
+		);
+	}
+
+	private function recacheJson() {
+		global $wgCreateWikiGlobalWiki;
+
+		$cWJ = new CreateWikiJson( $wgCreateWikiGlobalWiki );
+		$cWJ->resetDatabaseList();
```
Comment (JohnFLewis) [inline maintenance/changeDBCluster.php:64]: What about resetWiki as well?

Comment (paladox) [reply maintenance/changeDBCluster.php:72]: Thanks, done.

**Pull request merged**